utils/utils.go

```diff
@@ -259,10 +295,42 @@ func WatchCerts() {
 	}
 }
 
+// parseSSHOptions parses options from ssh.ParseAuthorizedKey format to our map format for SSHAuthKey.
+func parseSSHOptions(options []string) map[string][]string {
+	ret := make(map[string][]string)
+	for _, o := range options {
+		values := make([]string, 0)
+		optionSplit := strings.Split(o, "=")
+
+		if len(optionSplit) == 1 {
```

Comment:
Why do we expect this to happen?

Reply:
Some options from sshd(8) have no values. Most of them are illogical to sish, but this is how we can comply with the format. And possibly add some valueless options in the future

Reply:
bump